testing/common.hh

```diff
@@ -13,8 +13,10 @@ static const float pi2 = pi * 2.f;
 #include <gtest/gtest.h>
 #include <gmock/gmock.h>
 
+using ::testing::_;
 using ::testing::Contains;
 using ::testing::ElementsAre;
+//using ::testing::FieldsAre;
```

Comment:
Maybe os/compiler check so we don't have to comment things out/enable them

Reply:
Removed it.